package.json

```diff
@@ -10,6 +10,7 @@
   },
   "devDependencies": {
     "@octokit/core": "^3.4.0",
+    "@typescript-eslint/eslint-plugin": "^5.52.0",
```

Comment:
fwiw there was a new release yesterday but the release notes don't list anything relevant: https://github.com/typescript-eslint/typescript-eslint/releases/tag/v5.53.0